src/__experimental__/components/numeral-date/numeral-date.spec.js

```diff
@@ -256,4 +257,16 @@ describe('NumeralDate', () => {
       });
     });
   });
+
+  it('the required prop is passed to the input', () => {
+    wrapper = renderWrapper({ label: 'required', required: true, dateFormat: ['dd', 'mm', 'yyyy'] });
+
+    const inputs = wrapper.find('input');
+    inputs.forEach((input) => {
+      expect(input.prop('required')).toBe(true);
+    });
+
+    const label = wrapper.find(Label);
+    expect(label.prop('isRequired')).toBe(true);
+  });
```

Comment:
two tests in one case

Reply:
Updated test description because the setup is the same.